src/pzt/mysql/sections/10-try/10-certificate.mdx

```diff
@@ -0,0 +1,55 @@
+<p>
+  Before you can configure {props.pzt.name} TLS, you will need a certificate
+  issued by a trusted certificate authority (CA). If you already have a
+  certificate, private key, and CA root certificate from your organization's
+  existing CA, you can skip to the {props.pzt.name} TLS configuration section
+  below. If you need to generate a certificate, you can:
+</p>
+
+<ul>
+  <li>
+    Use a{' '}
+    <a href="/signup?product=cm">
+      free hosted Smallstep Certificate Manager authority
+    </a>
+  </li>
+  <li>
+    Roll your own with{' '}
+    <a href="/docs/step-ca">
+      open source <inlineCode>step-ca</inlineCode>
+    </a>
+  </li>
+</ul>
+
+<Alert severity="warning">
+  This tutorial is designed for private networks, not publicly trusted websites
+  or endpoints. If you are looking to automate certificates for public websites,
+  servers, or endpoints, we recommend using{' '}
+  <a href="https://letsencrypt.org/">Let's Encrypt</a> and the ACME instructions
+  below.
+</Alert>
+
+<p>
+  To request a certificate from your CA using the{' '}
+  <a href="https://smallstep.com/docs/step-cli">
+    <inlineCode>step</inlineCode>
+  </a>{' '}
+  CLI, bootstrap your CA with <inlineCode>step ca bootstrap</inlineCode> and run
+  the following command (sub the server name for the actual name / DNS name of
+  your {props.pzt.name} server).
+
+</p>
+
+<MDXBlock path="snippets/certificate" />
+
+Note that MySQL uses RSA keys, thus the `--kty=RSA` here.
+Your certificate and private key will be saved in `server-cert.pem` and `server-key.pem`, respectively.
+
+MySQL is one of a few services that require the intermediate CA certificate to be bundled with the root CA certificate, in order to validate certificate chains. Let's create a bundle of your CA intermediate and root certificates:
+
+```shell
+cat server-cert.pem | awk '/BEGIN/,/END    /{ if(/BEGIN/){a++}; if (a==2) {print} }' > ca.pem
+step ca root >> ca.pem
+```
```

Comment:
Could we use `<CodeBlock>` with the `copyText` param here so it doesn't copy two lines? Or would this run fine with the newline since it doesn't have the prompts?

Reply:
I'll convert it to a CodeBlock with copyText